README.md

```diff
@@ -112,6 +112,26 @@ and as such requires several environment variables to be set:
 * `AWS_REGION` should be set to, e.g., `us-east-1`.
 * `CLOUDWATCH_LOG_GROUP_NAME` should be set to a test log group name, which you may need to precreate.
 
+You may configure the above environment variables in your `~/.m2/settings.xml` and they will be applied automatically when running `mvn test`:
```

Comment:
Just making it a bit easier to run the tests, see also the changes in `pom.xml`.